### ISSUE TITLE
Hue Dimmer Switch

### ISSUE THREAD

**Issue by who93** [mozilla-iot/gateway#1472]
Hi,
I guess the Hue Dimmer Switch is not working properly, because it is not listed in the supported device page?

I managed to pair it and the gateway recognizes, if the "On" button is pressed and reports "1-pressed". However, the "Off" button and the two dimmer buttons are not recognized. Have you planned to support the dimmer switch?

Is it possible to add more advanced rules? E.g.: "1-pressed" and "Light"=on, turn off "Light" and "1-pressed" and "Light"=off, turn on "Light"? My goal is to toggle the light with the "On" button of the dimmer.

Thanks!

**Comment by mrstegeman** [comment]
I assume you're trying to do this through the Zigbee adapter?

**Comment by mrstegeman** [comment]
Relevant hue adapter PR: mozilla-iot/philips-hue-adapter#32

**Comment by who93** [comment]
yes, I am trying to do it with the Zigbee adapter. The linked PR is for the hue adapter?

**Comment by mrstegeman** [comment]
Yeah, we just had a similar issue for that adapter. I’m going to move this issue.

@dhylands Is there anything that would help you debug this?
